cluster/operations/datadog.yml

```diff
@@ -34,6 +34,24 @@
         # time (whoever starts up last)
         unique_friendly_hostname: true
 
+        # by default, the agent will emit disk usage for every volume.
+        # this results in noisy, high-cardinality metrics that are also
+        # very expensive! these two config flags override the defaults,
+        # and disable per-volume reporting.
+        generate_disk_config: false
+        disk_yaml_config: |
+          ---
+          init_config:
+
+          instances:
+            - use_mount: yes
+              tag_by_filesystem: true
+              all_partitions: true
+              device_blacklist:
+              - /var/vcap/data/worker/work/volumes/*/**
```

Comment:
ahh interesting. I tried to copy a pattern I saw in an email from a Datadog support person. But let's try the regex and see what happens - you want to do it or want me to? I might not have access to prod systems at some point very soon because I already sent off my offboarding request.

Reply:
oh true hahah np! I can take it 😁  thanks!

Reply:
damn, tried quite a bit, many of the rules I tried just didn't really work - we still get those measurements with all of the fields included 😞  I'll give it a try another time soon